src/containers/ToDoList/ToDoList.jsx

```diff
@@ -117,6 +134,18 @@ export const ToDoList = () => {
 
   const ThemeChangingIcon = isDarkTheme ? LightModeIcon : DarkModeIcon;
 
+  const handleSortDates = () => {
+    const sortedTasks = [...tasks].sort((a, b) => {
+      const dateA = new Date(a.dateAdded);
+      const dateB = new Date(b.dateAdded);
+
+      return dateA - dateB;
+    });
+
+    setTasks(sortedTasks);
+    saveTasksToCookies(sortedTasks);
+  };
+
```

Comment:
This should be best implemented in the same way you did filtering. So sorting is handled only on displaying phase.

The tasks list in state can remain untouched. It all contains all information you need. You use these information to render properly. We can cover this piece tomorrow.